deps/v8/tools/js2c.py

```diff
@@ -32,7 +32,7 @@
 # library.
 
 import os, re, sys, string
-import optparse
+import argparse
```

Comment:
See previous conversation about this here: https://github.com/nodejs/io.js/pull/2022#issuecomment-113622842